### PR TITLE
Ballistics rebalance: I looooove GitHub edition

### PR DESCRIPTION
**Pull request by maaacha**
## About The Pull Request
With the help of Daemon, the thunderdome, and plenty of cope analysis, I am happy to bring you a pretty heavy nerf/fix to a lot of the problems currently plaguing the ballistics weapons. Some weapons had the incorrect ammo typings, others had inexplicable damage bonuses, etc. A full list of changes is in the changelog.

Also I learned how to use GitKraken and fixed the commit issue.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
balance: Cowboy Repeater: Fire rate brought in line with other repeaters
balance: .357 Revolver: Fire rate brought in line with similar revolvers
balance: 50 MG: Fixed ammunition type error. 50MG is now surplus instead of match and improvised is now handloaded instead of surplus. Effectively nerfs AMR damage (98 -> 75). Increased range on AMR scope from 13 tiles to 18 tiles to compensate. AMR has been given 15% AP so it still fills it's niche of armour killer.
balance: Minigun - Increased recoil, brought in line with other 5mm automatic firearms. Negative damage modifier applied (30 -> 19.5)
balance: Service Rifle - Fire rate decreased, brought in line with similar low tier semi automatic rifles.
balance: Thompson SMG - Decreased fire rate, brought in line with similar mid tier SMGs
balance: Sniper Rifle - Lowered damage bonus from tier 4 to tier 3 (60.75 -> 56.25). Lowered scope range from 2x to 1.5x, Increased recoil to match similar high calibre long range firearms.
balance: 14mm -- Lowered base damage (50 -> 43) 
balance: Police revolver fire rate lowered from faster to fast.
balance: 22 pistol fire rate changed from 'as fast as you can click' to 'pretty fast'
balance: Increased R84 recoil and lowered fire rate to make it the 'low tier' LMG, instead of just being the LSW 2.0

/:cl:

### CODE REVIEW

**Comment by YosemiteYam** [comment]
RIP 14mm, you were a good boy, shredding death on your lonesome

**Comment by maaacha** [comment]
> RIP 14mm, you were a good boy, shredding death on your lonesome
> 
> Does the sniper rifle scope range match binoculars now? Cause the fact that it saw farther than binos was messed up

The sniper rifle uses a weird zoom that just applies a modifier to how far you can see. The base binoculars let you see an extra six tiles in the direction you're looking, and I've made it so the sniper lets you see half as far as you used to. Kind of hard to get a direct read on the tile distance without more extensive testing, if it's still oppressive after the nerf I'll squash the range even more.

File: code/modules/projectiles/guns/ballistic/minigun.dm [hunk 1]
```diff
@@ -99,9 +99,9 @@
 	autofire_shot_delay = GUN_AUTOFIRE_DELAY_FASTEST
 	burst_shot_delay = GUN_BURSTFIRE_DELAY_FASTEST
 	burst_size = 1
-	damage_multiplier = GUN_EXTRA_DAMAGE_0
+	damage_multiplier = GUN_LESS_DAMAGE_T4
```
Comment (Zero-Percent-Angel) [inline code/modules/projectiles/guns/ballistic/minigun.dm:102]: Seems fairly heavy on top of everything else

Comment (maaacha) [reply code/modules/projectiles/guns/ballistic/minigun.dm:102]: Minigun has 300rpm + normal rifle recoil, surplus 5mm does 30 damage by default and 38.75 with match.

**Comment by spockye** [comment]
If yer nerfing the minigun, include a nerf for the laser type aswell. Theres no reason to use the ballistic one as far as i understand it

**Comment by maaacha** [comment]
> If yer nerfing the minigun, include a nerf for the laser type aswell. Theres no reason to use the ballistic one as far as i understand it

Is the laser gatling even acquireable in game?

**Pull request merged**